CodeEditor/Sources/CodeEditor/UXCodeTextViewRepresentable.swift

```diff
@@ -48,40 +48,40 @@ public struct UXCodeTextViewRepresentable: UXViewRepresentable {
      *   - autoscroll:  If enabled, the editor automatically scrolls to the respective
      *                  region when the `selection` is changed programatically.
      */
-
+    
     public init(editorBindings: EditorBindings) {
         self.editorBindings = editorBindings
     }
     
-    private var editorBindings: EditorBindings
-
+    var editorBindings: EditorBindings
```

Comment:
Small suggestion: Maybe leave this private and instead create functions `setSelectedFeedbackSuggestionId` and `setShowAddFeedback` to then call those from `onLightBulbTap`.

Reply:
Good suggestion